commons/active/questionnaire/questionnaire.avsc

```diff
@@ -6,6 +6,7 @@
   "fields": [
     { "name": "time", "type": "double", "doc": "Timestamp in UTC (s) when the questionnaire is submitted to the subject." },
     { "name": "timeCompleted", "type": "double", "doc": "Timestamp in UTC (s) when subject marks the questionnaire as." },
```

Comment:
```suggestion
    { "name": "time", "type": "double", "doc": "Timestamp in UTC (s) when the questionnaire is started by the subject." },
    { "name": "timeCompleted", "type": "double", "doc": "Timestamp in UTC (s) when subject marks the questionnaire as completed." },
```
Doc update to avoid any ambiguities.